### DISCUSSION TITLE
Broken FPA coding in Cupra Ateca 2018 (help needed)

### DISCUSSION THREAD

**Discussion by PetrKodyrenkov** [jilleb/MQB-FPA#93] [Q&A]
Hi,
Seems i've somehow killed driving profile selection on my Cupra Ateca 12/2018 while playing with car scanner.
Driving profile selector does nothing, no lights are coming on on selected profile. also error is explored:
`19. CAN Gateway
DTCs: 1
01025 [0x000401]
B200500: Data record invalid ungültiger Datensatz, Funktion "Fahrprofilauswahl"
Status bits: 1 
Priority: 2 
Frequency: 1 
Unlearning index: 152 
Status: Test failed, Confirmed`
Maybe you have an idea how that could be fixed?
Probably by uploading FPA profile from compatible car?

**Comment by jilleb** [comment]
Yes that is probably the case. Playing with datasets can be risky. My fpa tool  on mqbtools.nl has a Cupra Ateca dataset that you can download and flash to your car with VCP, Odis or VCTool.

**Reply by PetrKodyrenkov** [reply]
Thank you for an answer, i've found 2 Ateca profiles on a website, both saying (2020) and my car is from 2018.
Also Developer of "Car Scanner" mentioned that "it seems that for your VIN code it should be "LP" dataset version."
I've checked "mqbtools.nl" and found several FPA's starting from "LP" but non of them seems to be for Ateca.
I'll try tomorrow flashing profiles from your amazing tool and will let you know what is the result.
But maybe you have some ideas about getting LP Dataset for ateca somewhere, so i could also check this?

p.s. Thank you for all the job you did, mqbtools.nl is amazing source of the info, thank you for gathering all that together and your researches on that topic.

**Comment by jilleb** [comment]
Thank you for the kind words.

Don't install the LPxx on your car, because they are from the newer models after 2022. Instead, just try TA00. It's from 2020, but that shouldn't hurt your car.

**Reply by PetrKodyrenkov** [reply]
TA00 is for Seat Ateca rather then Cupra Ateca(which is listed on your website as TB00 or TC00) according to the list 
You sure about TA00?

**Reply by jilleb** [reply]
Oh good one,  take the Cupra one.

**Reply by PetrKodyrenkov** [reply]
Car Scanner update with LP dataset was released before i was able to reach person with vcds, so i've flashed it and it seems to be working fine(i'll do more tests today) so i think i will just leave it as-is for now if it works, don't want to break it again.
Thank you for your support!
